explorer/explorer.go

```diff
@@ -92,6 +115,13 @@ func (e *explorerSvc) GetFeeRate() (float64, error) {
 	return response["1"], nil
 }
 
+func (e *explorerSvc) ListenAddresses(utxoHandler func(StreamUtxoUpdate) error) error {
+	if e.addrTracker == nil {
+		return fmt.Errorf("address tracker not initialized")
```

Comment:
If the websocket connection is not available, it means we have to start polling for the subscribed addresses (if not already done) instead of returning an error.

This function basically returns error only if the explorer returned error to us, otherwise a channel (or nil if we already listen) is always returned